Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7
+
+COPY . /securitybot
+
+env PYTHONPATH $PYTHONPATH:/securitybot
+
+RUN apt-get update && \
+    apt-get install -y mysql-client && \
+    pip install -r /securitybot/requirements.txt
+
+WORKDIR /securitybot
+
+ENTRYPOINT ["/securitybot/docker_entrypoint.sh"]
```

Comment:
That was dumb of me, didn't create the user properly, and hadn't tested rebuild properly.  Fixed now and added docker test to travis.